src/osemgrep/networking/Http_helpers.ml

```diff
@@ -1,52 +1,63 @@
-open Http_lwt_client
-
-(* The http-lwt-client package offers:
```

Comment:
Would have been good in this file to add a toplevel comment about the design decision
to choose cohttp over other libraries, especially to explain that we got
lots of DNS timeout and even though this was partially fixed by Robur
(see the blog post about it), we still got many DNS issues and so decided to switch
back to cohttp unfortunately.

Also might be worth adding some comments about what happened with yosef and the TLS and
ocaml-tls vs openssh.